diff --git a/CMakeLists.txt b/CMakeLists.txt
index 244f20d..639d28c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -9,7 +9,7 @@ project(${TARGET_NAME})
 
 #set(CMAKE_CXX_STANDARD 17)
 
-include_directories(src/include duckdb/third_party/httplib duckdb/extension/autocomplete/include)
+include_directories(src/include ${DUCKDB_MODULE_BASE_DIR}/third_party/httplib ${DUCKDB_MODULE_BASE_DIR}/extension/autocomplete/include)
 add_subdirectory(src)
 
 build_static_extension(${TARGET_NAME} ${EXTENSION_SOURCES})
